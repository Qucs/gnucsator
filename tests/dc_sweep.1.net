
options trace

Eqn:Eqn1 I1="1" Export="yes"
Eqn:Eqn2 R1="1" Export="yes"

Idc:I1 _net0 gnd I="I1"
.SW:SW1 Sim="DC1" Type="lin" Param="I1" Start="10m" Stop="20m" Points="2"
.SW:SW2 Sim="SW1" Type="lin" Param="R1" Start="10" Stop="30" Points="3"
.DC:DC1 Temp="26.85" reltol="0.001" abstol="1 pA" vntol="1 uV" saveOPs="no" MaxIter="150" saveAll="no" convHelper="none" Solver="CroutLU"
R:R1 _net0 gnd R="R1"
