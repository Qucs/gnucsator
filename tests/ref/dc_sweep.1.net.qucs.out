<Qucs Dataset 0.0.20>
<indep I1 2>
  +1.00000000000000002082e-02
  +2.00000000000000004163e-02
</indep>
<indep R1 3>
  +1.00000000000000000000e+01
  +2.00000000000000000000e+01
  +3.00000000000000000000e+01
</indep>
<dep _net0.V I1 R1>
  +9.99999999999999916733e-02
  +1.99999999999999983347e-01
  +1.99999999999999983347e-01
  +3.99999999999999966693e-01
  +2.99999999999999988898e-01
  +5.99999999999999977796e-01
</dep>
